src/js/rx/components/ErrorMessages.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const ErrorMessages = ({ errors }) => {
```

Comment:
If this is going to be used with Secure Messaging as while, it should probably be in `common`. Can be done when that becomes the case.